client/src/components/existing-request/names-gray-box.vue

```diff
@@ -50,28 +29,52 @@ export default class NamesGrayBox extends Vue {
   @Prop({ default: () => [] })
   readonly names: any[]
 
-  private getIcon (name): string {
+  private getTooltipText (name: any): string {
     switch (name.state) {
-      case NameState.APPROVED:
-      case NameState.CONDITION: return 'mdi-check'
+      case NameState.APPROVED: return 'Approved for use.'
+      case NameState.CONDITIONAL: return `Approved for use.`
+      case NameState.NOT_EXAMINED: return `Not reviewed. ${name.decision_text}`
+      case NameState.REJECTED: return `Rejected. ${name.decision_text}`
+      default: return ''
+    }
+  }
 
+  private getIcon (name:any): string {
+    switch (name.state) {
+      case NameState.APPROVED: return 'mdi-check'
+      case NameState.CONDITIONAL: return 'mdi-check'
+      case NameState.NOT_EXAMINED: return ''
       case NameState.REJECTED: return 'mdi-close'
-
-      case NameState.NE:
       default: return ''
     }
   }
 
-  private getClass (name): string {
+  private getClass (name: any): string {
     switch (name.state) {
-      case NameState.APPROVED:
-      case NameState.CONDITION: return 'approved'
-
+      case NameState.APPROVED: return 'approved'
+      case NameState.CONDITIONAL: return 'approved'
+      case NameState.NOT_EXAMINED: return ''
       case NameState.REJECTED: return 'rejected'
-
-      case NameState.NE:
       default: return ''
     }
   }
 }
 </script>
+
+<style lang="scss" scoped>
+#names-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex: 1 1 auto;
+}
+
+.name-container {
+  padding: 1.25rem 1rem;
+  cursor: help;
+}
+
+.tooltip {
+  width: unset !important;
+  max-width: 270px;
```

Comment:
I know this overrides the tooltip base class but it didn't look right with very little text (ie, "Approved for use.").

Reply:
Tracey said this is perfect. She wants a max width of 270px (except for some special tooltips).